### PR TITLE
Add a pause message for user

### LINKED ISSUE #1: Add message for pause
Add message for pause user

### PR DESCRIPTION
**Pull request by adprojectsstudio**
Resolve #1

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ﻿// See https://aka.ms/new-console-template for more information
 Console.WriteLine("Hello, World!");
+
+Console.WriteLine("Press any key to continue...");
```
Comment (angelo-p82) [inline Program.cs]: What happen if i press Enter?

Comment (adprojectsstudio) [reply Program.cs]: Nothing

Comment (angelo-p82) [inline Program.cs]: What if i press ESC?

Comment (angelo-p82) [inline Program.cs]: Change to "Press ENTER key..."

**Review by angelo-p82 — CHANGES_REQUESTED** [review]

**Review by angelo-p82 — APPROVED** [review]
OK!

**Pull request merged**